pkg/action/upgrade.go

```diff
@@ -398,30 +364,28 @@ func (u *Upgrade) releasingUpgrade(c chan<- resultMessage, upgradedRelease *rele
 		}
 	}
 
-	if u.Wait {
+	kubeClient, ok := u.cfg.KubeClient.(kube.ContextInterface)
+	if u.Wait && ok {
```

Comment:
Same deal w/ fallback here.

Reply:
Updated to maintain backwards compatability.